chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -257,7 +257,7 @@
           end
 
         when os =~ /^(open)?suse/
-          append << "install=#{install_url} autoyast=#{node_url}/autoyast.xml"
+          append << "install=#{install_url} autoyast=#{node_url}/autoyast-upgrade.xml autoupgrade=1"
```

Comment:
Yes of course. But the underlying mechanism required for that in crowbar are still being worked out (e.g. here: https://github.com/crowbar/barclamp-updater/pull/11) and implemented yet. This commit here contains just the bare minimum that we needed to create/test an AutoYaST profile for upgrading. It's not meant to be merged in this form (see my initial comment)